tests/test_bundle_trt_export.py

```diff
@@ -92,6 +94,42 @@ def test_trt_export(self, convert_precision, input_shape, dynamic_batch):
             self.assertTrue("meta_file" in json.loads(extra_files["def_args.json"]))
             self.assertTrue("network_def" in json.loads(extra_files["inference.json"]))
 
+    @parameterized.expand([TEST_CASE_3, TEST_CASE_4])
+    @unittest.skipUnless(has_onnx, "Onnx is required for onnx-trt conversion!")
+    def test_onnx_trt_export(self, convert_precision, input_shape, dynamic_batch):
+        meta_file = os.path.join(os.path.dirname(__file__), "testing_data", "metadata.json")
+        config_file = os.path.join(os.path.dirname(__file__), "testing_data", "inference.json")
+        with tempfile.TemporaryDirectory() as tempdir:
+            def_args = {"meta_file": "will be replaced by `meta_file` arg"}
+            def_args_file = os.path.join(tempdir, "def_args.yaml")
+
+            ckpt_file = os.path.join(tempdir, "model.pt")
+            ts_file = os.path.join(tempdir, f"model_trt_{convert_precision}.ts")
+
+            parser = ConfigParser()
+            parser.export_config_file(config=def_args, filepath=def_args_file)
+            parser.read_config(config_file)
+            net = parser.get_parsed_content("network_def")
+            save_state(src=net, path=ckpt_file)
+
+            cmd = ["python", "-m", "monai.bundle", "trt_export", "network_def", "--filepath", ts_file]
+            cmd += ["--meta_file", meta_file, "--config_file", f"['{config_file}','{def_args_file}']", "--ckpt_file"]
+            cmd += [ckpt_file, "--args_file", def_args_file, "--precision", convert_precision]
+            cmd += ["--use_onnx", "True"]
```

Comment:
Hi @Nic-Ma ,
     I have updated this PR according to the previous advices. As for this one, maybe we can add it after 1.2rc4 because the output of the conversion model and the original model may not be close in every ci test environment. And it could take a while pass the ci test.

Thank,
Bin